packages/web/src/components/nav/desktop/NavColumn.module.css

```diff
@@ -280,7 +280,7 @@ svg.logo path {
   /* negative margin so the link hover chip shows up. */
   margin-left: -16px;
   /* left padding is visibly (32px + -16px margin =) 16px to match the left padding of .groupHeader */
-  padding: 6px 0 6px 32px;
+  padding: 5px 0 5px 32px;
```

Comment:
how do we design system this 5px thing!!!!! 🤮

Reply:
`spacing(5/8)` 💩